capsules/src/lpm013m126.rs

```diff
@@ -339,6 +339,29 @@ where
         }
     }
 
+    fn uninitialize(&self) -> Result<(), ErrorCode> {
+        match self.state.get() {
+            State::Off => Err(ErrorCode::ALREADY),
+            _ => {
+                // TODO: clear pixels asynchronously, like the datasheet asks
+                // this stops extcomin timer
```

Comment:
I assume this is still work in progress, no? If this should be merged without the solved TODO, can you describe here the implications of not implementing this yet.

Reply:
This is not work in progress. It works as expected. I updated the comment.

Reply:
My assumption is that it works, but the display's datasheet suggests clearing all pixels? What about changing the `TODO` into `Improvement`?

Reply:
I'm not sure if it will actually be any improvement :P But I know it's something to investigate, and TODO reflects that well I think. Plus, convention says we grep code for "TODO", "FIXME", and "XXX" for rough edges.

Shall I still change it to "improvement"?

Reply:
It's now "TODO: investigate". I also added an empty line to separate the two comments.